--- a/debian/control	2026-02-23 13:15:16.114881165 +0100
+++ b/debian/control	2026-02-23 13:18:07.149042906 +0100
@@ -3,6 +3,7 @@
 Priority: optional
 Maintainer: Debian Libvirt Maintainers <pkg-libvirt-maintainers@lists.alioth.debian.org>
 Uploaders: Richard Jones <rjones@redhat.com>, Guido Günther <agx@sigxcpu.org>, Hilko Bengen <bengen@debian.org>
+Build-Conflicts: db-util
 Build-Depends: dpkg-dev (>= 1.22.5), debhelper-compat (= 13),
   po-debconf, bash-completion,
   supermin (>= 5.2.2-4~),
@@ -48,7 +49,6 @@
   libfuse-dev, fuse3 (>= 3.10.1-3) | fuse (<< 3),
   libglib2.0-dev, gjs [!s390x !armel] <!nocheck>, gobject-introspection, libgirepository1.0-dev,
   libgtk-3-dev,
-  db-util,
   netpbm, icoutils,
   po4a,
   flex, bison,
@@ -186,6 +186,7 @@
 Architecture: linux-any
 Multi-Arch: same
 Pre-Depends: ${misc:Pre-Depends}
+Conflicts: db-util
 Depends: ${shlibs:Depends}, ${misc:Depends},
   supermin (>= 5.2.2-4~),
   qemu-system-arm [armel armhf arm64],
@@ -197,7 +198,6 @@
   qemu-system-x86 [i386 amd64],
   qemu-system-misc [loong64],
   qemu-utils,
-  db-util,
   icoutils,
   netpbm,
   osinfo-db,
